lib/blockchainexplorers/v8.js

```diff
@@ -0,0 +1,422 @@
+  var client = this._getClient();
+  client.broadcast({ payload })
+    .then( (ret) => {
+console.log('[v8.js.218:ret:]',ret); //TODO
+      if (!ret.txid) {
+        return cb(new Error('Error broadcasting'));
+      }
+      return cb(null, ret.txid);
+    })
+    .catch(err => {
+console.log('[v8.js.221:err:]',err); //TODO
+        return cb(err);
+    });
+};
+
+V8.prototype.getTransaction = function(txid, cb) {
+  var self = this;
+console.log('[v8.js.207] GET TX', txid); //TODO
+  var client = this._getClient();
+  client.getTx({txid: txid })
+    .then( (tx) => {
+      if (!tx || _.isEmpty(tx)) {
+        return cb();
+      }
+      self.translateTx(tx);
+      return cb(null, tx);
+    })
+    .catch((err) =>{ 
+      // The TX was not found
+      if (err.statusCode == '404') {
+        return cb();
+      } else {
+        return cb(err);
+      }
+      
+    });
+};
+
+V8.prototype.getTransactions = function(wallet, startBlock, endBlock , cb) {
+console.time('getTxs');
+console.log(`getTXS: start /end ${startBlock}/${endBlock}`);
+   var self = this;
+
+   var client = this._getAuthClient(wallet);
+  var acum = '', broken;
+
+  var txStream = client.listTransactions({
+    pubKey: wallet.beAuthPublicKey,
+    payload: {
+    },
+    startBlock: startBlock,
+    endBlock: endBlock,
+//    startDate: 0,
+//    endDate: '2019-01-01',
+  });
+
+  txStream.on('data', (raw) => {
+    acum = acum + raw.toString();
+  });
+
+  txStream.on('end', () => {
+    if (broken)  {
+      return;
+    }
+
+    var txs = [], err;
+    _.each(acum.split(/\r?\n/), (rawTx) => {
+     if (!rawTx)
+      return;
+
+      var tx;
+      try {
+        tx = JSON.parse(rawTx);
+      } catch (e) {
+        log.error('v8 error at JSON.parse:' + e  + ' Parsing:' + rawTx + ":");
+        return cb(e);
+      }
+      if (tx.address && self.addressFormat) {
+        tx.address = self.translateResultAddresses(tx.address);
+      }
+      // v8 field name differences
+      if (tx.value)
+        tx.amount = tx.satoshis / 1e8;
+
+      txs.push(tx);
+    })
+    txs= _.flatten(txs);
+console.timeEnd("getTxs");
+    return cb(null, txs);
+  });
+
+  txStream.on('error', (e) => {
+    log.error('v8 error:' +  e );
+    broken = true;
+    return cb(e);
+  });
+}
+
+
+V8.prototype.getAddressActivity = function(address, cb) {
+  var self = this;
+
+  var url = this.baseUrl + '/address/' + this.translateQueryAddresses(address) + '/txs?limit=1';
+console.log('[v8.js.328:url:] CHECKING ADDRESS ACTIVITY',url); //TODO
+  request.get(url, {})
+    .then( (ret) => {
+      return cb(null, ret !== '[]');
+    })
+      .catch((err) => {
+console.log('[v8.js.335:err:]',err); //TODO
+        return cb(err);
+      } );
+};
+
+V8.prototype.estimateFee = function(nbBlocks, cb) {
+  var self = this;
+  nbBlocks = nbBlocks || [1,2,6,24];
+
+  var result = {};
+
+  async.each(nbBlocks, function(x, icb) {
+    var url = self.apiPrefix + '/fee/' + x;
```

Comment:
When creating a tx proposal with v8, there's an error about the fee estimation URL being invalid. I think this line should use `baseUrl`, like so:

```suggestion
    var url = self.baseUrl + '/fee/' + x;
```

After changing it, at least something is happening. But it's still not working, as it times out somewhere, even though the API response looks fine, and subsequently throws a `Error: Lock timeout (300000) exceed`.